storage/transfer_service/aws_request.py

```diff
@@ -31,45 +34,75 @@ def main():
     # Edit this template with desired parameters.
     # Specify times below using US Pacific Time Zone.
     transfer_job = '''
-    {
-        "description": "YOUR DESCRIPTION",
+    {{
+        "description": "{description}",
         "status": "ENABLED",
-        "projectId": "YOUR_PROJECT_ID",
-        "schedule": {
-            "scheduleStartDate": {
-                "day": 1,
-                "month": 1,
-                "year": 2015
-            },
-            "scheduleEndDate": {
-                "day": 1,
-                "month": 1,
-                "year": 2015
-            },
-            "startTimeOfDay": {
-                "hours": 0,
-                "minutes": 0
-            }
-        },
-        "transferSpec": {
-            "awsS3DataSource": {
-                "bucketName": "YOUR_SOURCE_BUCKET",
-                "awsAccessKey": {
-                    "accessKeyId": "YOUR_ACCESS_KEY_ID",
-                    "secretAccessKey": "YOUR_SECRET_ACCESS_KEY"
-                }
-            },
-            "gcsDataSink": {
-                "bucketName": "YOUR_SINK_BUCKET"
-            }
-        }
-    }
-    '''
+        "projectId": "{project_id}",
+        "schedule": {{
+            "scheduleStartDate": {{
+                "day": {day},
+                "month": {month},
+                "year": {year}
+            }},
+            "scheduleEndDate": {{
+                "day": {day},
+                "month": {month},
+                "year": {year}
+            }},
+            "startTimeOfDay": {{
+                "hours": {hours},
+                "minutes": {minutes}
+            }}
+        }},
+        "transferSpec": {{
+            "awsS3DataSource": {{
+                "bucketName": "{source_bucket}",
+                "awsAccessKey": {{
+                    "accessKeyId": "{access_key}",
+                    "secretAccessKey": "{secret_access_key}"
+                }}
+            }},
+            "gcsDataSink": {{
+                "bucketName": "{sink_bucket}"
+            }}
+        }}
+    }}
+    '''.format(description=description, project_id=project_id, day=day,
+               month=month, year=year, hours=hours, minutes=minutes,
+               source_bucket=source_bucket, access_key=access_key,
+               secret_access_key=secret_access_key, sink_bucket=sink_bucket)
 
     result = storagetransfer.transferJobs().create(body=json.loads(
         transfer_job)).execute()
     logging.info('Returned transferJob: %s', json.dumps(result, indent=4))
+# [END main]
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser(
+        description='Create a one-off transfer from Amazon S3 to GCS.')
+    parser.add_argument('description', help='Your description')
+    parser.add_argument('project_id', help='Your Google Cloud project ID')
+    parser.add_argument('date', help='Date YYYY/MM/DD')
```

Comment:
Be a bit more descriptive here, e.g., "Scheduled date for transfer."

Also, probably a good idea to make it optional and default to utcnow.